salt/modules/zypper.py

```diff
@@ -2359,7 +2494,7 @@ def list_provides(**kwargs):
     return ret
 
 
-def resolve_capabilities(pkgs, refresh, **kwargs):
+def resolve_capabilities(pkgs, refresh=False, root=False, **kwargs):
```

Comment:
Why sometimes `root` is `None` and sometimes `False`? This should be consistent.

Reply:
@meaksh is that OK to default `refresh` here?